task.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      
+      <meta charset="UTF-8">
+      <meta name="viewport"  ,initial-scale=1.0 >
+      <link rel="stylesheet" href="task.css">
+    <title>TASK </title>
+  </head>
+  <body>
+    <div class="container">
+        <div class="first">
+    <h1>  Remit  </h1><br><br><br>
+    <h2> Welcome </h2>
+    <p>  Sign in by entering the information below  </p><br><br>
+      <div class="wrap">  
+        <div class="in">
+             
+    <label  > Email Address</label>

```

Comment:
use `for` attr in label element
read more from here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for